data/external/scrapers/nat.py

```diff
@@ -42,9 +39,43 @@ def scrape_rooms():
         key = room["room_code"]  # needed, as room_code is removed in _sanitise_room
         rooms[key] = _sanitise_room(room)
 
+    # split the orgs into a different file
+    _extract_orgs(rooms)
+    for room in rooms.values():
+       org_ids = [room.pop("org")["org_id"]]
+       org_ids.extend(org["org"]["org_id"] for org in room.pop("orgs"))
+       room["org_ids"] = org_ids
     return rooms
 
 
+@cached_json("orgs_nat.json")
+def _extract_orgs(rooms: dict) -> dict:
+    """
+    Extract the organisations from the room information.
+    """
+    org_set: set[str] = set(json.dumps(room["org"], sort_keys=True) for room in rooms.values())
+    for room in rooms.values():
+        org_set.update(json.dumps(org["org"], sort_keys=True) for org in room["orgs"])
```

Comment:
```suggestion
    org_set: set[str] = set()
    for room in rooms.values():
        # Converting it to a JSON string makes it hashable
        org_set |= json.dumps(room["org"], sort_keys=True)
        org_set.update(json.dumps(org["org"], sort_keys=True) for org in room["orgs"])
```

Reply:
```suggestion
    org_set: set[str] = set()
    for room in rooms.values():
        # Converting dicts to a JSON strings makes them hashable
        org_set.update(json.dumps(room["org"], sort_keys=True)
        org_set.update(json.dumps(org["org"], sort_keys=True) for org in room["orgs"])
```

Reply:
> org_set.update(json.dumps(room["org"], sort_keys=True)

It seems that it needs to be an iterable of objects to update, so either

```python
org_set.add(json.dumps(room["org"], sort_keys=True))
```
or 
```python
org_set.update({json.dumps(room["org"], sort_keys=True)})
```

Reply:
I like the first suggestion more, what do you think?

Reply:
Yes, it's the method that you would normally use afaik